cognite/neat/_session/_session.py

```diff
@@ -38,10 +38,10 @@ def __init__(
         self.physical_data_model = PhysicalDataModel(self._store, self._client, self._config)
         self.issues = Issues(self._store)
         self.result = Result(self._store)
-        self.opt = Opt(self._store)
 
-        if self.opt._collector.can_collect:
-            self.opt._collector.collect("initSession", {"mode": self._config.modeling.mode})
+        collector = Collector()
+        if collector.can_collect:
```

Comment:
I think it is still useful to have the `can_collect` as it enables us to not log pytest run for example.